### DISCUSSION TITLE
requesting page number selector under tables

### DISCUSSION THREAD

**Discussion by JohnnyG-github** [mattwright324/youtube-metadata#140] [Ideas]
Hi, first of all i'm quite indebted to you for making "Youtube Metadata Bulk" and sharing it with us for use as it helped me immensely for documenting various videos across various channels and many thanks for that; though i'd like to ask for a feature that is "page number selector" below those tables;

for e.g. here i'm on page 87 but i want to go to page 108 directly, currently it seems there is no way to just select random page number from available pool and go to there directly and one has to manually press next next to reach desired page.
![image](https://user-images.githubusercontent.com/43844080/234810878-76dc6c7f-d796-4290-9673-ca5ca90e83a8.png)

i tried to use developer tool to feed a page number directly but i'm technically not that good with these stuff so couldn't do so,
so please if you have some time and if it's possible can you please add a 'page number selector' using drop down or anything with this project as it would be of immense help for people like me.

**Comment by mattwright324** [comment]
Hi @JohnnyG-github 

This is a good idea, I can see if DataTables supports this or has a plugin available to do so.
